components/button/stories/button.test.js

```diff
@@ -1,109 +1,130 @@
-import { Variants } from "@spectrum-css/preview/decorators";
-import { html } from "lit";
-import { capitalize } from "lodash-es";
+import { ArgGrid, Container, Variants } from "@spectrum-css/preview/decorators";
 import { Template } from "./template.js";
 
 /**
  * Multiple button variations displayed in one story template.
  * Used as the base template for the stories.
  */
-const CustomButton = ({ iconName, iconSet, ...args }, context) => html`
-	<div style="display: flex; column-gap: 13px; row-gap: 24px;">
-		${Template({
+const ButtonContentGroup = ({ iconName, iconSet, ...args }, context) => Container({
+	level: 3,
+	withBorder: false,
+	content: [
+		Template({
 			...args,
 			iconName: undefined,
-		}, context)}
-		${Template({
+		}, context),
+		Template({
 			...args,
 			iconName: iconName ?? "Edit",
 			iconSet: iconSet ?? "workflow",
-		}, context)}
-		${Template({
+		}, context),
+		Template({
 			...args,
 			hideLabel: true,
 			iconName: iconName ?? "Edit",
 			iconSet: iconSet ?? "workflow",
-		}, context)}
-	</div>
-`;
+		}, context),
+	],
+});
+
+const ButtonIconGroup = (args, context) => Container({
+	level: 3,
+	withBorder: false,
+	content:[
+		{
+			testHeading: "Workflow icon",
+			content: Template({
+				...args,
+				iconName: "Edit",
+				iconSet: "workflow",
+			}, context),
+		},
+		{
+			testHeading: "Workflow icon",
+			content: Template({
+				...args,
+				iconName: "Link",
+				iconSet: "workflow",
+			}, context),
+		},
+		{
+			testHeading: "UI icon",
+			content: Template({
+				...args,
+				// Uses a UI icon that is smaller than workflow sizing, to test alignment:
```

Comment:
The most minor of nits but I'd be inclined to remove `:` as we can assume the comment pertains to the line(s) that follow. ✨